.codecov.yml

```diff
@@ -13,3 +13,6 @@ coverage:
       default:
         informational: false
         target: 100%
+
+ignore:
```

Comment:
I think that as long as this is ignored in .coveragerc you don't need extra ignore configuration in codecov.

Reply:
Tried that; didn't work…